src/utils/login.ts

```diff
@@ -15,10 +16,17 @@ export async function autoLogin(
   try {
     const func = router["login"];
     if (!func) {
-      info("No login callback exists!");
-      sendMessageToTestBox(LOGIN_FAIL);
+      window.__tbxLoginEvent = {
+        testbox: {
+          event: "login",
+          data,
+          version: 1,
+          sender: MessageSender.APP,
+        },
+      };
       return;
```

Comment:
Relevant to add a timeout here that raises the `LOGIN_FAIL` after a certain time if the event is not consumed.

We can use 10 seconds for now.